src/views/TabView/TabBarBottom.js

```diff
@@ -39,6 +39,7 @@ type Props = {
   getLabelText: (scene: TabScene) => string;
   renderIcon: (scene: TabScene) => React.Element<*>;
   style: any;
+  labelStyle: any?;
```

Comment:
sorry, but I don't have any experience with flow yet.  I just copied that code from `TabBarTop`